### PR TITLE
feat: upgrade off-pg to 0.9 api and rename to vanilla-postgresql in release 0.9

### PR DESCRIPTION
**Pull request by kizuna-lek**
upgrade official-postgresql to 0.9 api and rename to vanilla-postgresql in release 0.9

### CODE REVIEW

File: addons/vanilla-postgresql/templates/cmpd-12.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  scripts:
+    - name: vanilla-postgresql-scripts
+      templateRef: {{ include "vanilla-postgresql.scriptsTemplate" . }}
+      namespace: {{ .Release.Namespace }}
+      volumeName: scripts
+      defaultMode: 0555
+  runtime:
+    {{- include "vanilla-postgresql.spec.runtime.common" . | nindent 4 }}
+    containers:
+      - image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major12.minor15 }}
```
Comment (shanshanying) [inline addons/vanilla-postgresql/templates/cmpd-12.yaml:28]: shouldn't image be set by cmpv?

File: addons/vanilla-postgresql/templates/cmpd-14.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  scripts:
+    - name: vanilla-postgresql-scripts
+      templateRef: {{ include "vanilla-postgresql.scriptsTemplate" . }}
+      namespace: {{ .Release.Namespace }}
+      volumeName: scripts
+      defaultMode: 0555
+  runtime:
+    {{- include "vanilla-postgresql.spec.runtime.common" . | nindent 4 }}
+    containers:
+      - image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major14.minor07 }}
```
Comment (shanshanying) [inline addons/vanilla-postgresql/templates/cmpd-14.yaml:28]: dito

**Review by shanshanying — COMMENTED** [review]
updates examples as well. (and there is no ClusterVersion, pls update accordingly)

**Comment by kizuna-lek** [comment]
> updates examples as well. (and there is no ClusterVersion, pls update accordingly)

/done

**Pull request merged**